cg/services/orders/store_order_services/store_case_order.py

```diff
@@ -34,263 +38,182 @@ def __init__(
-                    sample=sample,
-                )
+                    self._update_relationship(
+                        father=sample_father,
+                        link=case_sample,
+                        mother=sample_mother,
+                        sample=sample,
+                    )
             self.status_db.session.add_all(new_cases)
             self.status_db.session.add(status_db_order)
             self.status_db.session.commit()
         return new_cases
 
     @staticmethod
-    def _update_case_panel(panels: list[str], case: Case) -> None:
+    def _update_case_panel(panels: list[str], case: DbCase) -> None:
         """Update case panels."""
         case.panels = panels
 
     @staticmethod
-    def _append_ticket(ticket_id: str, case: Case) -> None:
+    def _append_ticket(ticket_id: str, case: DbCase) -> None:
         """Add a ticket to the case."""
         case.tickets = f"{case.tickets},{ticket_id}"
 
     @staticmethod
-    def _update_action(action: str, case: Case) -> None:
+    def _update_action(action: str, case: DbCase) -> None:
         """Update action of a case."""
         case.action = action
 
     @staticmethod
-    def _update_relationship(father_obj, link_obj, mother_obj, sample):
-        link_obj.status = sample["status"] or link_obj.status
-        link_obj.mother = mother_obj or link_obj.mother
-        link_obj.father = father_obj or link_obj.father
-
-    def _create_link(self, case_obj, family_samples, father_obj, mother_obj, sample):
-        link_obj = self.status_db.relate_sample(
-            case=case_obj,
-            sample=family_samples[sample["name"]],
-            status=sample["status"],
-            mother=mother_obj,
-            father=father_obj,
+    def _update_relationship(
+        father: DbSample | None, link: CaseSample, mother: DbSample | None, sample: SampleInCase
+    ) -> None:
+        link.status = getattr(sample, "status", link.status)
+        link.mother = mother or link.mother
+        link.father = father or link.father
+
+    def _create_link(
+        self,
+        case: DbCase,
+        family_samples: dict,
+        father: DbSample,
+        mother: DbSample,
+        sample: SampleInCase,
+    ) -> CaseSample:
+        link = self.status_db.relate_sample(
+            case=case,
+            sample=family_samples[sample.name],
+            status=getattr(sample, "status", None),
+            mother=mother,
+            father=father,
         )
-        self.status_db.session.add(link_obj)
-        return link_obj
+        self.status_db.session.add(link)
+        return link
 
-    def _create_sample(self, case, customer_obj, order, ordered, sample, ticket):
-        sample_obj = self.status_db.add_sample(
-            name=sample["name"],
-            comment=sample["comment"],
-            control=sample["control"],
-            internal_id=sample["internal_id"],
-            order=order,
+    def _create_sample(
```

Comment:
just for clarity and consistency with the other services
```suggestion
    def _create_db_sample(
```